src/man/kiwix-serve.1

```diff
@@ -1,102 +1,89 @@
-Exit if the specified process ID (\fBPID\fR) is not running anymore.
+\fB--library\fR
+By default, kiwix-serve expects a list of ZIM files as command line arguments. Providing the --library option tells kiwix-serve that the command line argument is rather a library XML file.
 
 .TP
-\fB-d\fR, \fB--daemon\fR
-Detach the HTTP server daemon from the main process.
+\fB-h, --help\fR
+Print a help message.
 
 .TP
-\fB-i\fR, \fB--address\fR \fBADDRESS\fR
-Listen only on the specified IP address (\fBADDRESS\fR). If not provided, it listens on all available IP addresses.
+\fB-i ADDR, --address=ADDR\fR
+Listen only on this IP address. By default, the server listens on all available IP addresses.
 
 .TP
-\fB-M\fR, \fB--monitorLibrary\fR
-Monitor the XML library file and automatically reload it when changes are detected.
+\fB-p PORT, --port=PORT\fR
+TCP port on which to listen for HTTP requests (default: 80).
 
 .TP
-\fB-m\fR, \fB--nolibrarybutton\fR
-Don't print the built-in home button in the built-in top bar overlay.
+\fB-r ROOT, --urlRootLocation=ROOT\fR
+URL prefix on which the content should be made available (default: empty).
 
 .TP
-\fB-n\fR, \fB--nosearchbar\fR
-Don't print the built-in bar overlay on the top of each served page.
+\fB-d, --daemon\fR
+Detach the HTTP server daemon from the main process.
 
 .TP
-\fB-b\fR, \fB--blockexternal\fR
-Prevent users from directly accessing external links.
+\fB-a PID, --attachToProcess=PID\fR
+Exit when the process with id PID stops running.
 
 .TP
-\fB-p\fR, \fB--port\fR \fBPORT\fR
-Specify the TCP port on which the server should listen to HTTP requests. The default port is 80.
+\fB-M, --monitorLibrary\fR
+Monitor the XML library file and reload it automatically when it changes.
 
 .TP
-\fB-r\fR, \fB--urlRootLocation\fR \fBURL_ROOT_LOCATION\fR
-Specify the URL prefix on which the content should be made available. The default is '/'.
+\fB-m, --nolibrarybutton\fR
+Disable the library home button in the ZIM viewer toolbar.
 
 .TP
-\fB-s\fR, \fB--searchLimit\fR \fBSEARCH_LIMIT\fR
-Specify the maximum number of ZIM files in a full-text multi-zim search. The default is no limit.
+\fB-n, --nosearchbar\fR
+Disable the search box in the ZIM viewer toolbar.
 
 .TP
-\fB-t\fR, \fB--threads\fR \fBTHREADS\fR
-Specify the number of threads to run in parallel. The default is 4.
+\fB-b, --blockexternal\fR
+Prevent users from directly navigating to external resources via links in ZIM content.
 
 .TP
-\fB-v\fR, \fB--verbose\fR
-Print debug log to STDOUT.
+\fB-t N, --threads=N\fR
+Number of threads to run in parallel (default: 4).
 
 .TP
-\fB-V\fR, \fB--version\fR
-Print the software version.
+\fB-s N, --searchLimit=N\fR
+Maximum number of ZIM files in a fulltext multizim search (default: No limit).
 
 .TP
-\fB-z\fR, \fB--nodatealiases\fR
-Create URL aliases for each content by removing the date.
+\fB-z, --nodatealiases\fR
+Create URL aliases for each content by removing the date embedded in the file name.
```

Comment:
The description of this option in the html documentation comes with an example. The absence of that example here makes the effect of this option difficult to understand for a first time user.